packages/components/templates/f-template-subnav/CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v0.7.0
+------------------------------
+*June 23, 2022*
+### Changed
+- Bumped devDependencies to prevent issue with conflicting versions of dependancy peerDeps
+
+### Changed
+- Updated `f-navigation-links` to v1.3.0.
+
```

Comment:
A few spacing nits, also I think you've duplicated a bit of the changelog from the previous entry, probably from a merge conflict

```suggestion
v0.7.0
------------------------------
*June 24, 2022*

### Changed
- Bumped devDependencies to prevent issue with conflicting versions of dependency peerDeps


```